sysml/src/examples/Geometry Examples/VehicleGeometryAndCoordinateFrames.sysml

```diff
@@ -1,33 +1,77 @@
 package VehicleGeometryAndCoordinateFrames {
+    import TrigFunctions::*;
+    import ISQ::*;
 	import SI::*;
 	import Time::*;
 
     import ShapeItems::*;
     import SpatialItems::*;
 
+    import UnitsAndScales::CoordinateFrame;
     import UnitsAndScales::TranslationRotationSequence;
     import UnitsAndScales::Translation;
     import UnitsAndScales::Rotation;
     
     private import Collections::Array;
+    private import ScalarValues::Boolean;
     private import ScalarValues::Real;
+    private import ScalarValues::Natural;
+	private import ControlFunctions::forAll;
+    
+    part def Vehicle :> CompoundSpatialItem;
 
-    part def Vehicle :> SpatialItem;
+    part def Chassis :> CompoundSpatialItem {
+        item :>> shape = Box(4800 [mm], 1840 [mm], 1350 [mm]);
+    }
 
-    part def Wheel :> SpatialItem {
+	/**
+	 * Generic wheel with lugbolts
+	 * 
+	 * The radius is estimated for a 22 inch hub plus 110 mm tire height.
+	 * The wheel width is equal to the cylinder height.
+	 * The wheel has 5 lugbolts that are evenly distributed along a circle centered at the wheel's center.
+	 */
+    part def Wheel :> CompoundSpatialItem {
```

Comment:
Something should not be a `CompoundSpatialItem` unless it actually has at least one `componentItem`, because a `CompoundSpatialItem` is _made up_ of its `componentItems`. I now think `componentItems` really needs to have multiplicity 1..* to make this clear.